Backpack/Toast/Classes/BPKToast.m

```diff
@@ -128,12 +131,19 @@ - (void)hudWasHidden:(MBProgressHUD *)hud {
     }
 }
 
+#pragma mark - Private methods
+
 - (void)setupHUD {
     self.hud.bezelView.style = MBProgressHUDBackgroundStyleSolidColor;
     self.hud.bezelView.backgroundColor = [[BPKColor dynamicColorWithLightVariant:BPKColor.skyGray darkVariant:BPKColor.blackTint03] colorWithAlphaComponent:0.85];
     self.hud.contentColor = BPKColor.white;
     self.hud.delegate = self;
     self.hud.accessibilityIdentifier = ToastAccessibilityIdentifier;
+    self.hud.accessibilityViewIsModal = YES;
+}
+
+- (void)postShowAccessibilityNotification {
+    UIAccessibilityPostNotification(UIAccessibilityLayoutChangedNotification, self.hud.label);
```

Comment:
I'm not sure whether we should send focus to the toast, or just announce it leaving focus intact.
Sending focus to the toast could be a poor experience if it disappears automatically. eg what happens if someone is still hearing the content when it disappears? Where will focus go to after it does.

Reply:
Yeah I was concerned about that too. Doing it this way would be a breaking change to introduce an accessibility label

Reply:
We could easily adopt such a breaking change in the app by just combining the title and description. We can inform squads of this and ask them to fix if necessary?
It would still be better than what we currently have

Reply:
Yeah fair